packages/components/pie-thumbnail/src/thumbnail.scss

```diff
@@ -1 +1,28 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+// Base thumbnail styles
+.c-thumbnail {
+  --thumbnail-size-small: var(--dt-spacing-g);
+  --thumbnail-border-radius-small: var(--dt-radius-rounded-b);
+  --thumbnail-bg-color: var(--dt-color-container-default);
+  --thumbnail-border-outline: 1px solid var(--dt-color-border-default);
+
+  &.c-thumbnail--outline {
+    --thumbnail-border: var(--thumbnail-border-outline);
+  }
+
+  box-sizing: border-box;
+  width: var(--thumbnail-size-small);
+  height: var(--thumbnail-size-small);
+  border-radius: var(--thumbnail-border-radius-small);
+  border: var(--thumbnail-border);
+  background-color: var(--thumbnail-bg-color);
+}
+
+.c-thumbnail-img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+  display: block;
+  border-radius: var(--thumbnail-border-radius-small);
+}
```

Comment:
nit: this could be nested within `.c-thumbnail` so all of the styles are encapsulated